cirq/protocols/trace_distance_bound.py

```diff
@@ -83,7 +104,17 @@ def _strat_distance_from_unitary(val: any) -> Optional[float]:
             return 0.0
         return squared**0.5
 
-    angles = np.sort(np.angle(np.linalg.eigvals(u)))
+    return trace_distance_from_angle_list(np.angle(np.linalg.eigvals(u)))
+
+
+def trace_distance_from_angle_list(angle_list: Iterable[float]) -> float:
+    """ Given a list of arguments of the eigenvalues of a unitary matrix,
+    calculates the trace_distance_bound of the unitary effect.
+
+    The maximum provided angle should not exceed the minimum provided angle
+    by more than two pi.
```

Comment:
Use LaTeX or unicode for pi.  Most of the docstrings in Cirq use `π`.

Reply:
swapped to symbol